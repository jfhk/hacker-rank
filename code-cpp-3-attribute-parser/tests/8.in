6 5
<tag1 value = "Hello World" okay = "">
<tag2 name = "Name 1">
<tag3 attr = "">
</tag3>
</tag2>
</tag1>
tag1.tag2~name
tag1~name
tag1~value
tag1~okay
tag1.tag2.tag3~attr
