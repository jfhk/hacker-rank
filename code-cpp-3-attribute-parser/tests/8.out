Name 1
Not Found!
Hello World


